decidim-surveys/db/migrate/20200609090533_check_legacy_tables.rb

```diff
@@ -1,34 +1,55 @@
 # frozen_string_literal: true
 
-namespace :decidim_surveys do
-  desc "Migrate data from decidim_surveys tables to decidim_forms tables"
-  task migrate_data_to_decidim_forms: :environment do
-    class Answer < ApplicationRecord
-      self.table_name = :decidim_surveys_survey_answers
-    end
+# rubocop:disable Rails/Output
+# rubocop:disable Style/GuardClause
+# rubocop:disable Lint/UnreachableCode
+class CheckLegacyTables < ActiveRecord::Migration[5.2]
+  class Answer < ApplicationRecord
+    self.table_name = :decidim_surveys_survey_answers
+  end
 
-    class AnswerChoice < ApplicationRecord
-      self.table_name = :decidim_surveys_survey_answer_choices
-    end
+  class AnswerChoice < ApplicationRecord
+    self.table_name = :decidim_surveys_survey_answer_choices
+  end
 
-    class AnswerOption < ApplicationRecord
-      self.table_name = :decidim_surveys_survey_answer_options
-    end
+  class AnswerOption < ApplicationRecord
+    self.table_name = :decidim_surveys_survey_answer_options
+  end
 
-    class Question < ApplicationRecord
-      self.table_name = :decidim_surveys_survey_questions
-    end
+  class Question < ApplicationRecord
+    self.table_name = :decidim_surveys_survey_questions
+  end
+
+  def up
+    return if Rails.env.test? || ENV["CI"] == "true"
 
-    unless [Answer, AnswerChoice, AnswerOption, Question].all? { |model| ActiveRecord::Base.connection.table_exists? model.table_name }
-      puts "ERROR: There are not all the necessary surveys tables. Have you already migrated the data?"
-      next
+    if tables_exists.any?
+      puts "If you already migrated the data, the following raise statement can be safely removed and migrations can continue to be run again."
+      puts "But beware, they will remove some surveys' legacy tables!"
+      puts "Otherwise check this migration in order to keep your data safe. Legacy tables will be removed by the following migrations."
+      raise "ERROR: there's the risk to loose legacy information from old surveys!"
+
+      if tables_exists.all?
+        migrate_legacy_data if Question.any?
+      else
+        puts "Some legacy surveys tables exist but not all. Have you already migrated the data?"
+        puts "Migrate or backup your data and then remove the following raise statement to continue with the migrations (that will remove surveys legacy tables)"
```

Comment:
Please also add 

```bash
puts "For migrating your data you can do that with the command:" 
puts "bundle exec rake decidim_surveys:migrate_data_to_decidim_forms"
```

Reply:
@andreslucena  I've removed the task as requested in the related issue and copied its contents into the migration.
Maybe we can keep the task for this release and remove it in a future release?

Reply:
Ah sorry, my mistake.

>  Maybe we can keep the task for this release and remove it in a future release? 

Ok